tagging/taggers.py

```diff
@@ -22,16 +22,18 @@ def _get_env_variable(container: Container, variable: str) -> str:
     raise KeyError(variable)
 
 
+_PIP_VERSION_PREFIX = "Version: "
```

Comment:
I think this belongs within the function, it isn't something reused elsewhere anyhow right? If it's in this bigger scope, I'd assume it was.

Reply:
Ok, I think the best thing to do is to just revert this.